NAMESPACE

```diff
@@ -41,6 +45,7 @@ export(harmonize_twas)
 export(lasso_weights)
 export(lbf_to_alpha)
 export(load_LD_matrix)
+export(load_colocboost_regional_data)
```

Comment:
this, and the colocboost pipeine function are the only two that should be exported. Perhaps not others